Main.py

```diff
@@ -93,6 +93,8 @@ def main(args, seed=None, baked_server_options: Optional[Dict[str, object]] = No
                         f"Location IDs: {min(cls.location_id_to_name):{numlength}} - "
                         f"{max(cls.location_id_to_name):{numlength}}")
 
+    AutoWorld.call_stage(world, "can_generate")
```

Comment:
i would expect a can_ method to return a boolean rather than raise an exception

Reply:
yeah, this might be more of an assert_generate or raises_generate_issues

Reply:
Ok, will change to `assert_generate`